draft-ietf-acme-acme.md

```diff
@@ -547,6 +547,7 @@ in the "type" field (within the "urn:ietf:params:acme:error:" namespace):
 | Type                    | Description                                                                    |
 |:------------------------|:-------------------------------------------------------------------------------|
 | accountDoesNotExist     | The request specified an account that does not exist                           |
+| alreadyRevoked          | The certificate has already been revoked                                       |
```

Comment:
I think this description would might match the others a little more closely if it were phrased along the lines of "The request specified a certificate to be revoked that has already been revoked"